python/taichi/main.py

```diff
@@ -88,6 +92,48 @@ def __call__(self):
 
         return getattr(self, args.command)(sys.argv[2:])
 
+    @staticmethod
+    def _check_version():
+        # Check Taichi version for the user.
+        print('Checking your Taichi version...')
+        major = _ti_core.get_version_major()
+        minor = _ti_core.get_version_minor()
+        patch = _ti_core.get_version_patch()
+        version = f'{major}.{minor}.{patch}'
+        payload = {'version': version, 'platform': '', 'python': ''}
+
+        system = platform.system()
+        if system == 'Linux':
+            payload['platform'] = 'manylinux1_x86_64'
+        elif system == 'Windows':
+            payload['platform'] = 'win_amd64'
+        elif system == 'Darwin':
+            if platform.release() < '19.0.0':
+                payload['platform'] = 'macosx_10_14_x86_64'
+            elif platform.machine() == 'x86_64':
+                payload['platform'] = 'macosx_10_15_x86_64'
+            else:
+                payload['platform'] = 'macosx_11_0_arm64'
+
+        python_version = platform.python_version()
+        if python_version.startswith('3.6'):
+            payload['python'] = 'cp36'
+        elif python_version.startswith('3.7'):
+            payload['python'] = 'cp37'
+        elif python_version.startswith('3.8'):
+            payload['python'] = 'cp38'
+        elif python_version.startswith('3.9'):
+            payload['python'] = 'cp39'
+
+        response = requests.post('http://54.90.48.192/check_version',
```

Comment:
Maybe hold on merging this PR until the IP gets a TLS+DNS, random ip address scares ppl :P

Reply:
What about the address that contains 'aws' and 'amazon'?

Reply:
That may be better, but still a bit awkward, but we can update it later.